File: app/src/main/java/coolpharaoh/tee/speicher/tea/timer/views/newtea/suggestions/NoSuggestions.java
package coolpharaoh.tee.speicher.tea.timer.views.newtea.suggestions;

class NoSuggestions implements Suggestions {

    @Override
    public int[] getAmountTsSuggestions() {
        return new int[]{};
    }

    @Override
    public int[] getAmountGrSuggestions() {
        return new int[]{};
    }

    @Override
    public int[] getTemperatureCelsiusSuggestions() {
        return new int[]{};
    }

    @Override
    public int[] getTemperatureFahrenheitSuggestions() {
        return new int[]{};
    }

    @Override
    public String[] getTimeSuggestions() {
        return new String[]{};
    }
}

File: app/src/main/java/coolpharaoh/tee/speicher/tea/timer/views/new_tea/suggestions/NoSuggestions.kt
package coolpharaoh.tee.speicher.tea.timer.views.new_tea.suggestions

class NoSuggestions : Suggestions {

    override val amountTsSuggestions: IntArray
        get() = intArrayOf()

    override val amountGrSuggestions: IntArray
        get() = intArrayOf()

    override val amountTbSuggestions: IntArray
        get() = intArrayOf()

    override val temperatureCelsiusSuggestions: IntArray
        get() = intArrayOf()

    override val temperatureFahrenheitSuggestions: IntArray
        get() = intArrayOf()

    override val timeSuggestions: Array<String>
        get() = arrayOf()
}